sdk/agrifood/agrifood-farming-rest/review/agrifood-farming.api.md

```diff
@@ -7926,7 +7892,7 @@ export interface ScenesListQueryParamProperties {
     endDateTime?: Date | string;
     imageFormats?: string;
     imageNames?: string;
-    imageResolutions?: Array<number>;
+    imageResolutions?: string;
```

Comment:
It should be a breaking, in this case, the swagger definition is defined as array of numbers with a multi collection format https://github.com/Azure/azure-rest-api-specs/blob/e38daec67d57ef9c4804b1e3055753407e45fa71/specification/agrifood/data-plane/Microsoft.AgFoodPlatform/preview/2022-11-01-preview/agfood.json#L22132-L22142. the new generation should be expected. However, codegen should have supported it at that time, the problem is why this was not changed by then? 
Not sure if it's service team edit the generated code?